website/content/en/preview/tasks/globalsettings.md

```diff
@@ -62,6 +62,9 @@ data:
   # The VM memory overhead as a percent that will be subtracted 
   # from the total memory for all instance types 
   aws.vmMemoryOverheadPercent: "0.075"
+  # Interruption Handling is currently in BETA and is disabled by default. Enabling interruption handling may
```

Comment:
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ are enabled in beta. I'd call this one alpha, tbh.